scraper_data.py

```diff
@@ -33,5 +39,13 @@ def main(counties: Tuple[str,...], output:str) -> None:
     else:
         print(json.dumps(out,indent=2))
 
+    if not out: # all counties failed
+        return 126
```

Comment:
In some shells, 126 and 127 have special meanings, so this one might not be a great choice. Other than that, things are pretty free-form and the shell won’t do anything special with the code, so 1-125 is usually safe.

(If you really want to finesse it, many \*nix systems define some common codes in `/usr/include/sysexits.h`, which you can check out. There’s not much out there that really treats programs using those specially, though. It’s more of a loose standard.)

Reply:
Went with `70` as it seems appropriate for a complete failure: https://www.freebsd.org/cgi/man.cgi?query=sysexits&apropos=0&sektion=0&manpath=FreeBSD%204.3-RELEASE&format=html